lib/websocket_server.rb

```diff
@@ -5,7 +5,7 @@ def initialize
     @pairing = {}
     @sockets = Concurrent::Array.new
 
-    Thread.new do
+    @transmitter = Thread.new do
```

Comment:
Maybe here or in a followup PR, you might want to create a `start_transmitter_thread` method that does this.
Add attr_readers for the two ivars above.  Again, not for this PR but something to consider if it makes sense.

```ruby
def start_transmitter_thread
  @transmitter ||= begin
    Thread.new do
    ...
    end
  end
end
```